index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <a class="logo" href="#">Rubanza Mark</a>
+        <nav>
+          <a class="menu" href="#"
+            ><img src="img/IconMenu.png" alt="IconMenu"
+          /></a>
+        </nav>
+      </header>
+      <main>
+        <section class="mainContainer">
+          <div class="informativeText">
+            <h1>Hey There.<br />I'm Mark</h1>
+            <h2>I am a Software Developer</h2>
+            <p>
+              I can help you build a product, feature or<br />website Look through
+              some of my work<br />and experience! If you like what you see<br />and
+              have a project you need coded,<br />
+              don’t hestiate to contact me.
+            </p>
+          </div>
+          <div class="socialIcons">
+            <ul class="socials">
+              <li>
+                <a href="https://github.com/banzaman">
+                  <img src="img/IconGitHub.png" alt="-GitHub button" />
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://www.linkedin.com/in/mark-rubanza-anderson-4399a2211/"
+                >
+                  <img src="img/IconLinkedin.png" alt="Linkedin button" />
+                </a>
+              </li>
+              <li>
+                <a href="https://angel.co/u/mark-rubanza-anderson">
+                  <img src="img/IconAngelist.png" alt="Angelist button" />
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/banzamarq10">
+                  <img src="img/IconTwitter.png" alt="Twitter button" />
+                </a>
+              </li>
+              <li>
+                <a href="https://medium.com/">
+                  <img src="img/IconMedium.png" alt="Medium button" />
+                </a>
```

Comment:
Kindly add the images folder because all the images do not appear here too.

 Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/153925529-2ed6d471-bd16-4af1-9cc4-39719131ed9b.png) | ![image](https://user-images.githubusercontent.com/35535130/153925662-4b05e78a-de1c-4ec6-a1c3-9820f7908a0f.png) |

Reply:
All good now 👍🏽